[Update] Wikis #2 (#237)
NilPointer-Software/bdfd-wiki: src/bdscript/startThread.md

# $startThread
Creates a new thread in the provided channel.

## Syntax
```
$startThread[Name;Channel ID;Message ID;(Archive duration;Return thread/channel ID)]
```

### Parameters
- `Name`: The name of the newly created thread.
- `Channel ID`: The channel where the thread shall be created.

> - `Message ID`: The orginal message that the thread is created for, can be left empty.
> - `Archive duration`: The duration when the thread should be auto-archived. Accepts `60` (1 hour), `1440` (1 day), `4320` (3 days), or `10080` (7 days) as input. Defaults to `60` (Optional).

Reviewer: ```suggestion
- `Message ID`: The message from which the thread will be created. Can be left empty.
- `Archive duration`: The duration after which the thread will be auto-archived due to inactivity. Accepts `60` (1 hour), `1440` (1 day), `4320` (3 days), or `10080` (7 days) as input. Defaults to `60` (Optional).
```
